CODE_OF_CONDUCT.md

```diff
@@ -69,12 +69,11 @@ dispute. If you are unable to resolve the matter for any reason, or if the
 behavior is threatening or harassing, report it. We are dedicated to providing
 an environment where participants feel welcome and safe.
 
-Reports should be directed to balintp@google.com, the Project Steward for 
-Cirq. It is the Project Steward’s duty to receive and address reported 
-violations of the Code of Conduct. They will then work with a committee 
-consisting of representatives from the Open Source Programs Office and the 
-Google Open Source Strategy team. If for any reason you are uncomfortable 
-reaching out to the Project Steward, please email opensource@google.com.
+Reports should be directed to [quantumai-oss-maintainers@googlegroups.com](mailto:quantumai-oss-maintainers@googlegroups.com),
```

Comment:
Nit - GitHub seems to detect and activate email links in markdown and rst without special syntax.  Please adjust here and below.  Temporary example: https://github.com/quantumlib/Cirq/blob/40832d819322777e851e8a1d7cd22ccd2d3832c8/CODE_OF_CONDUCT.md#conflict-resolution

Also, let us keep the 80-character line wrap for the sake of those who read text files as text.  :-)

```suggestion
Reports should be directed to quantumai-oss-maintainers@googlegroups.com,
```

Reply:
I guess I never noticed that before. Thanks for the tip!

Regarding 80 chars: in fact, I try to do that always, but in this case, the README.rst file already had those long lines, and I didn't want to format it in order to keep this PR only about the address changes. I'm working on converting the files to Markdown and am about to do a PR for that, and there, the results will be ≤ 80 chars.

Reply:
@pavoljuhas I could, however, reformat the README.rst file to keep to 80 chars, if that's preferred?

Reply:
@mhucka - apologies about the confusion - I meant the CODE_OF_CONDUCT.md which is within the 80 char limit in the main, so it would be nice to keep such format after the PR.

No need to worry about README.rst if it already has long lines.  I agree this can be left for later, especially if you already plan on converting to markdown.  (we also don't want to have a lot of extra diffs beyond the subject of the PR)

Reply:
OK, at first I thought I couldn't break the mailto hyperlinks across lines in the CoC file, but I just checked, and yes it is. So you're right, I should have done that.

That's no longer needed with the change to avoid hyperlinking, but I'll know to do that in the future.